chart/compass/charts/instance-creator/values.yaml

```diff
@@ -0,0 +1,19 @@
+deployment:
+  minReplicas: 1
+  maxReplicas: 2
+  autoscaling:
+    targetCPUUtilizationPercentage: 80
+    targetMemoryUtilizationPercentage: 75
+  image:
+    pullPolicy: IfNotPresent
+  resources: {}
+  resourcesIstioProxy:
+    requests: {}
+    limits: {}
+  args:
+    containerPort: 8080
+  securityContext: # Set on container level
+    runAsUser: 2000
```

Comment:
Do we need specifically user 2000? Or this is more or less not to run as root?

Reply:
I followed the pattern of the other charts. TBH, I don't remember why we came up with that user initially.